tests/ui/large_futures.fixed

```diff
@@ -1,7 +1,7 @@
 #![warn(clippy::large_futures)]
 #![allow(clippy::never_loop)]
 #![allow(clippy::future_not_send)]
-#![allow(clippy::manual_async_fn)]
+#![allow(clippy::manual_async_fn, clippy::uninlined_format_args)]
```

Comment:
Please be consistent. Either use one `#![allow]` per line, or group all `#![allow]` into the same invocation.